### PR TITLE
global: review of cookiecutter in preparation for beta releases

### PR DESCRIPTION
**Pull request by lnielsen**

### CODE REVIEW

File: scripts/repocheck.py [hunk 1]
```diff
@@ -0,0 +1,195 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Cookiecutter - Invenio Module Template
+# Copyright (C) 2016 CERN
```
Comment (jirikuncar) [inline scripts/repocheck.py]: `, 2017`

File: {{ cookiecutter.project_shortname }}/README.rst [hunk 2]
```diff
@@ -18,10 +18,9 @@
...
         :target: https://github.com/{{ cookiecutter.github_repo }}/blob/master/LICENSE
 
-
-{{ cookiecutter.description }}
-
 *This is an experimental developer preview release.*
 
-* Free software: GPLv2 license
-* Documentation: https://{{ cookiecutter.project_shortname }}.readthedocs.io/
+TODO: Please provide feature overview of module
```
Comment (jirikuncar) [inline {{ cookiecutter.project_shortname }}/README.rst]: Why are we not including also `{{ cookiecutter.description }}`?

Comment (lnielsen) [inline {{ cookiecutter.project_shortname }}/README.rst]: The idea is that each module should provide a more detailed description of its features (i.e. some few bullets) - see e.g. [Invenio-Admin's README.rst](https://github.com/inveniosoftware/invenio-admin) for an example.

Comment (jirikuncar) [inline {{ cookiecutter.project_shortname }}/README.rst]: We can keep the TODO, but for some simple modules the description is perfectly fine for the beginning.

Comment (lnielsen) [inline {{ cookiecutter.project_shortname }}/README.rst]: Fixed

**Comment by lnielsen** [comment]
Merged #91 into PR as well.

**Pull request merged**